parmed/gromacs/gromacstop.py

```diff
@@ -10,6 +10,10 @@
 from datetime import datetime
 import math
 import os
+try:
+    import pwd
+except ImportError:
+    pwd = None
```

Comment:
man, that `pwd` stuff was fixed a while ago and I just fork my repo, not `ParmEd/ParmEd` repo.